content/secrets/04-walkthrough-step-01/copilot.md

```diff
@@ -0,0 +1,25 @@
+---
+title: "Embedded tab content"
+disableToc: true
+hidden: true
+---
+
+In this section, it is important to match the names as described in order for the tutorial to work (so that it matches the format in the repository)
+
+Application Name: `ecsworkshop`
+
+Workload Type: `Load Balanced Web Service`
+
+Service Name: `todo-app`
+
+Deploy local test environment: `yes`
+
+
+```bash
+copilot init
+```
+
+{{< figure src="/images/secrets-copilot-init.gif" alt="Secrets Diagram" width="800px" >}}
+
+This will import values in the project located in `copilot/todo-app/manifest.yml`
```

Comment:
This is a great time to explain what copilot is doing, and walk through the various items it is creating on the users behalf. 

Also would love a brief explainer on what the manifest is, and how to read more about it with a link to the copilot docs.

Reply:
Similar to how we do it here: https://ecsworkshop.com/microservices/frontend/

Reply:
I see we are building an Aurora Serverless instance, this is another great opp to talk about the addons directory and what we're creating.